commit 84d5cbd190a2601a5deea0758345b2bcdd583ebf (refs/changes/02/2502/2)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2017-09-07T15:12:14+00:00 (2 years, 1 month ago)
    
    OS-6322 Fix zone device policy file
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/overlay/generic/etc/security/device_policy b/overlay/generic/etc/security/device_policy
new file mode 100644
index 00000000..88d6afa0
--- /dev/null
+++ b/overlay/generic/etc/security/device_policy
@@ -0,0 +1,28 @@
+#
+# Device policy configuration file.   When devices are opened the
+# additional access controls in this file are enforced.
+#
+# The format of this file is subject to change without notice.
+#
+# Default open privileges, must be first entry in the file.
+#
+
+*		read_priv_set=none		write_priv_set=none
+bridge:*	read_priv_set=net_rawaccess	write_priv_set=net_rawaccess
+fssnap:ctl	read_priv_set=sys_config	write_priv_set=sys_config
+icmp:*		read_priv_set=net_icmpaccess	write_priv_set=net_icmpaccess
+icmp6:*		read_priv_set=net_icmpaccess	write_priv_set=net_icmpaccess
+ipf:*		read_priv_set=sys_ip_config	write_priv_set=sys_ip_config
+ip:*		read_priv_set=net_rawaccess	write_priv_set=net_rawaccess
+ip6:*		read_priv_set=net_rawaccess	write_priv_set=net_rawaccess
+ipnet:*		read_priv_set=net_observability	write_priv_set=net_observability
+ipsecah:*	read_priv_set=sys_ip_config	write_priv_set=sys_ip_config
+ipsecesp:*	read_priv_set=sys_ip_config	write_priv_set=sys_ip_config
+keysock:*	read_priv_set=sys_ip_config	write_priv_set=sys_ip_config
+mm:allkmem	read_priv_set=all		write_priv_set=all
+mm:kmem		read_priv_set=none		write_priv_set=all
+mm:mem		read_priv_set=none		write_priv_set=all
+openeepr:*					write_priv_set=all
+random:*					write_priv_set=sys_devices
+scsi_vhci:devctl				write_priv_set=sys_devices
+spdsock:*	read_priv_set=sys_ip_config	write_priv_set=sys_ip_config
